### ISSUE TITLE
All Python 3 builds failing on Travis. 

### ISSUE THREAD

**Issue by carltongibson** [encode/django-rest-framework#6546]
[Since this build](https://travis-ci.org/encode/django-rest-framework/builds/512449691)

```
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/tox_venv/hooks.py", line 99, in tox_testenv_create
    venv.session.make_emptydir(venv.path)
AttributeError: 'VirtualEnv' object has no attribute 'session'
```

`<poem>`Sigh. CI.`</poem>`

**Comment by tomchristie** [comment]
A new version of tox-venv presumably?
We could pin tox and tox-venv in the travis file.

**Comment by rpkilby** [comment]
I'm currently working on an update to tox-venv. The quick fix is to pin `tox==3.7.0`.

**Comment by tomchristie** [comment]
Sounds legit.

**Comment by tomchristie** [comment]
Quick fix FTW

**Comment by carltongibson** [comment]
Latest build passed... (will have a look later.)

**Comment by rpkilby** [comment]
Closing as this should be fixed now. Released tox-venv v0.4.0

**Comment by rpkilby** [comment]
btw, we could actually just remove the tox-venv dependency. There isn't any benefit in using it here, since we don't have a warnings build (a la django-filter).
